articles/azure/azs-how-create-key-vault-portal.md

```diff
@@ -0,0 +1,106 @@
+
+## Prerequisites
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the UKCloud Azure Stack portal.
+
+## Creating a new Key Vault
+
+1. Log in to the UKCloud Azure Stack portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. In the favourites panel, select **Create a resource**.
+
+    ![New option in favourites panel](images/azsp_newmenu.png)
+
+3. In the *New* blade, select **Security + Identity**.
+
+   ![Security option in New blade](images/azs-browser-new-security.png)
+
+4. In the *New* blade, select **Key Vault**.
+
+   ![New Key Vault](images/azs-browser-new-key-vault.png)
+
+5. In the **Create key vault** blade, enter the following information:
+
+   - **Name** - The name of the key vault.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+   - **Access policies** - Configure who can access the key vault and what permissions they have.
+
+     ![Create new key vault](images/azs-browser-create-key-vault.png)
+
+6. Click **Create**.
+
+7. You can monitor the progress of your Key Vault's deployment by clicking the **Notifications** icon.
+
+   ![!Notification showing vault deployment in progress](images/azsp_createvm_progress.png)
+
+## Adding a secret to the Key Vault
+
+1. Once the key vault has been deployed, navigate to it by clicking **All services** in the favourites panel, then selecting **Key Vaults** under the *Security + Identity* section.
+
+2. Select your key vault from the list.
+
+3. In the **Settings** section of the key vault blade, select **Secrets**.
+
+4. On the **Secrets** page, click the **+ Generate/Import** button.
+
+5. In the **Create a secret** blade, enter the following information:
+
+   - **Upload Options** - Manually enter a secret or upload an x.509 certificate.
+
+   - **Name** - The name of the secret.
+
+   - **Value** - The value you are storing as a secret.
+
+   - **Content Type** - The type of content contained in the secret (For example, Password). There are no pre-defined values for this field.
+
+   - **Activation Date** - Sets when the secret will become active.
+
+   - **Expiration Date** - Sets when the secret will become inactive.
+
+   - **Enabled** -Set whether or not the secret data can be retrieved.
+
+   ![Create a new secret](images/azs-browser-create-secret.png)
+
+6. Click **Create**.
+
+## Viewing the secret
+
+1. Once the secret has been created, select it from the **Secrets** page.
+
+2. On the secret blade, select the current version.
+
+3. On the current version blade, click the **Show secret value** button.

```

Comment:
Check the case of the blade title and match how it is in the UI. And put it in italics.

Reply:
Think it should be current version's? Not sure about italics in this case?
![image](https://user-images.githubusercontent.com/41685901/46526276-b81e5000-c885-11e8-96f4-b0b62be10be5.png)